searchhotel.html

```diff
@@ -0,0 +1,303 @@
+      </li>
+
+      <li class="option-description">
+        <a class="option-description-link" href="#">
+          <img class="option-description-image" src="" alt="">
+          <h3 class="option-description-title">Desert Quail Inn</h3>
+        </a>
+        <img class="option-description-stars" src="" alt="">
+        <p class="option-description-rating">Рейтинг: 8,9</p>
+        <p class="option-description-designation">Мотель</p>
+        <b class="option-description-price">От 3000 ₽</b>
+        <a href="#" class="option-description-button button">Подробнее</a>
+        <a href="#" class="option-description-button button-favorites">В избранное</a>
+      </li>
+
+      <li class="option-description">
+        <a class="option-description-link" href="#">
+          <img class="option-description-image" src="" alt="">
+          <h3 class="option-description-title">Villas at Poco Diablo</h3>
+        </a>
+        <img class="option-description-stars" src="" alt="">
+        <p class="option-description-rating">Рейтинг: 9,2</p>
+        <p class="option-description-designation">Апартаменты</p>
+        <b class="option-description-price">От 2000 ₽</b>
+        <a href="#" class="option-description-button button">Подробнее</a>
+        <a href="#" class="option-description-button button-elected">В избранном</a>
+        <!--Здесь достаточно изменить класс, чтобы через стили изменить визуал? Или вообще это стоит делать через JS, чтобы при клике данные менялись в модалке "Все избранные"-->
+      </li>
+
+      <li class="option-description">
+        <a class="option-description-link" href="#">
+          <img class="option-description-image" src="" alt="">
+          <h3 class="option-description-title">GreenTree Inn</h3>
+        </a>
+        <img class="option-description-stars" src="" alt="">
+        <p class="option-description-rating">Рейтинг: 9,2</p>
+        <p class="option-description-designation">Гостиница</p>
+        <b class="option-description-price">От 3000 ₽</b>
+        <a href="#" class="option-description-button button">Подробнее</a>
+        <a href="#" class="option-description-button button-favorites">В избранное</a>
+      </li>
+    </ul>
+    <button class="button pagination-load-more" type="button">Загрузить ещё</button>
+    <ul class="pagination">
+      <li class="pagination-item">
+        <a class="pagination-link pagination-current">1</a>
+      </li>
+      <li class="pagination-item">
+        <a class="pagination-link" href="#">2</a>
+      </li>
+      <li class="pagination-item">
+        <a class="pagination-link" href="#">3</a>
+      </li>
+      <li class="pagination-item">
+        <a class="pagination-link" href="#">4</a>
+      </li>
+      <li class="pagination-item">
+        <a class="pagination-link pagination-all">
+          <a class="pagination-link" href="#">...</a>
+          <!--Не уверена, как правильно оформить вариант с многоточием-->
+        </a>
+      </li>
```

Comment:
Я думаю одного `a` хватит
```suggestion
      <li class="pagination-item">
        <a class="pagination-link pagination-all">…</a>
      </li>
      <li class="pagination-item">
        <a class="pagination-link" href="#">10</a>
      </li>
```